src/Containers/FileUpload/FileUpload.tsx

```diff
@@ -466,8 +482,20 @@ export const FileUpload: React.FC<IFileUploadProps> = ({
                         );
                     }
                     if (base64StringFile) {
-                        setBase64(base64StringFile);
+                        setIsSuccess(true)
+                        setIsFailure(false)
+                        setIsUploading(false)
+                        setTimeout(()=>{
+                            if(isMounted.current)
+                                setIsSuccess(false)
+                            setBase64(base64StringFile)
+                        },messageDuration)
```

Comment:
is this only SuccessMessageDuration or includes errors as well?

Reply:
includes both isSuccess and isFailure message.